src/service/url-expander/expander.js

```diff
@@ -134,6 +134,9 @@ export class Expander {
           if (!ignoringChars) {
             ignoringChars = true;
             nextArgShouldBeRaw = true;
+            user().assert(builder === '' || builder === ' ',
```

Comment:
`builder.trim() === ''`

Reply:
do you think we should allow n number of whitespaces? I was trying to prevent this: `REPLACE(hello-world,                `-`)`

Reply:
It's currently allowed.

Reply:
good point.

Reply:
done